src/components/Header/Header.jsx

```diff
@@ -62,7 +62,7 @@ export const Header = props => {
   const canSeeProjectManagementTab = props.hasPermission('seeProjectManagement') || props.hasPermission('seeProjectManagementTab');
   const canPostProject = props.hasPermission('postProject');
   // Tasks
-  const canUpdateTask = props.hasPermission('updateTask');
+  const canUpdateTask = props.hasPermission('updateTask') || props.auth.user?.permissions?.frontPermissions.includes('updateTask');
```

Comment:
```suggestion
  const canUpdateTask = props.hasPermission('updateTask');
```
Invividual permissions are checked through `props.hasPermission()` in addition to role permissions.